src/pretalx/orga/templates/orga/cfp/question_form.html

```diff
@@ -15,6 +15,9 @@
         {% bootstrap_field form.variant layout='horizontal' %}
         {% bootstrap_field form.help_text layout='horizontal' %}
         {% bootstrap_field form.required layout='horizontal' %}
+        {% if action != 'create' %}
+            {% bootstrap_field form.active layout='horizontal' %}
```

Comment:
given the impact, a "deactivate"/"activate"-button would be nicer here.

Reply:
Moved to #246